pkg/security/secl/model/model.go

```diff
@@ -715,6 +715,78 @@ func (f *FileFields) GetInUpperLayer() bool {
 	return f.Flags&UpperLayer != 0
 }
 
+// HashState is used to prevent the hash resolver from retrying to hash a file
+type HashSate int
+
+const (
```

Comment:
we could use https://pkg.go.dev/golang.org/x/tools/cmd/stringer that we already use here : https://github.com/DataDog/datadog-agent/blob/main/pkg/security/config/dump.go#L6

Reply:
done 👍🏻